lib/apollo-federation/schema.rb

```diff
@@ -47,7 +47,7 @@ def to_graphql
         super
       end
 
-      def federation_sdl(context: nil)
+      def federation_sdl(context:)
```

Comment:
Looks like I broke this when attempting to make this backwards compatible, context: https://github.com/Gusto/apollo-federation-ruby/pull/45#discussion_r370944253

I think if we default to `nil` the following way, we don't need to update `service_field.rb`

```suggestion
      def federation_sdl(context = nil)
```